wolfcrypt/src/poly1305.c

```diff
@@ -228,11 +229,11 @@ extern void poly1305_final_avx2(Poly1305* ctx, byte* mac);
     }
 
     static void U32TO8(byte *p, word32 v) {
-        p[0] = (v      ) & 0xff;
-        p[1] = (v >>  8) & 0xff;
-        p[2] = (v >> 16) & 0xff;
-        p[3] = (v >> 24) & 0xff;
-    }
+    p[0] = (byte)((v      ) & 0xff);
```

Comment:
Check indent here. Should be over 4 spaces (like it was).

Reply:
@dgarske, Done